unraid_mcp/tools/docker.py

```
@@ -264,56 +304,58 @@ async def unraid_docker(
         if action == "network_details" and not network_id:
             raise ToolError("network_id is required for 'network_details' action")
 
-        try:
+        if tail_lines < 1 or tail_lines > _MAX_TAIL_LINES:
+            raise ToolError(f"tail_lines must be between 1 and {_MAX_TAIL_LINES}, got {tail_lines}")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)